src/wazuh_db/wdb_fim.c

```diff
@@ -515,6 +515,13 @@ int wdb_fim_insert_entry2(wdb_t * wdb, const cJSON * data) {
         return -1;
     }
 
+    item_version = cJSON_GetStringValue(cJSON_GetObjectItem(data, "version"));
+
+    if (item_version == NULL) {
+        merror("DB(%s) fim/save request with no version attribute.", wdb->id);
```

Comment:
Does this mean that we will have an error for each sync attempt from older agents?